spec/models/vm/retirement_management_spec.rb

```diff
@@ -170,24 +162,19 @@
 
   it "#raise_retirement_event without current user" do
     event_name = 'foo'
-    event_hash = {:vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm",
-                  :retirement_initiator => "system"}
-    options = {:zone => @vm.my_zone}
+    event_hash = {:userid => nil, :vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm"}
 
-    expect(MiqEvent).to receive(:raise_evm_event).with(@vm, event_name, event_hash, options).once
+    expect(MiqEvent).to receive(:raise_evm_event).with(@vm, event_name, event_hash, nil).once
 
     @vm.raise_retirement_event(event_name)
   end
 
   it "#raise_retirement_event with current user" do
-    user = FactoryGirl.create(:user_with_group, :userid => 'freddy')
     event_name = 'foo'
-    event_hash = {:vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm",
-                  :retirement_initiator => "user", :userid => 'freddy'}
-    options = {:zone => @vm.my_zone}
+    event_hash = {:userid => nil, :vm => @vm, :host => @vm.host, :type => "ManageIQ::Providers::Vmware::InfraManager::Vm"}
```

Comment:
the title of this test says "with current user" so I was expecting a user to pass through here.